java/libphonenumber/src/com/google/i18n/phonenumbers/MultiFileMetadataSourceImpl.java

```diff
@@ -110,13 +110,18 @@ private boolean isNonGeographical(int countryCallingCode) {
   static <T> PhoneMetadata loadMetadataFromFile(
       T key, ConcurrentHashMap<T, PhoneMetadata> map, String filePrefix,
       MetadataLoader metadataLoader) {
+    // The size of the byte buffer used for deserializing the phone number metadata files for each
+    // region.
```

Comment:
You could define it at class level as a private final int constant.

Also do you know why it is this size, 16 *1024?